### DISCUSSION TITLE
T3S3 no Lora

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#22732] [Q&A]
Hi, i did compile for [LoRa32 V2.1_1.6](https://lilygo.cc/products/lora3) before and it worked.

Now i got [T3 S3](https://lilygo.cc/products/t3s3-v1-0).
But i don't get LoRa running.
 ´loraconfig´ is unkown.

i compiled with:
```
#ifndef USE_SPI 
#define USE_SPI                 // Add support for SPI
#endif

#define USE_SPI_LORA            // Add support for LoRaSend and LoRaCommand (+4k code)
#define USE_LORA_SX126X         // Add driver support for LoRa on SX126x based devices like LiliGo T3S3 Lora32 (+16k code)
```

with different compile (env) parameter.

i compiling with 14.3.0.4

What have I forgotten?

**Comment by sfromis** [comment]
You did not mention whether you made sure to also configure the relevant gpio pins, nothing will be "known" until that. Nor what build environment you used, the word "different" is really uninformative.

**Comment by ChrisVonFrosta** [comment]
which build envirement you would recomended? i tried many. For example: tasmota32, tasmota32s3, tasmota32-bluetooth.

![grafik](https://github.com/user-attachments/assets/2865d59d-82d3-4188-bd73-66292d85dc66)
![grafik](https://github.com/user-attachments/assets/75e21d02-502b-4478-9a41-cc12e1b16ec6)


i started with vsc to compile and ended with online compiler.

**Reply by sfromis** [reply]
With ESP32-S3, you should only use build environments for ESP32-S3, not for ESP32. That means names with `tasmota32s3`. It is rarely a good plan to just shoot at anything in sight....

What you posted indicates that you did **not** configure the required gpios.
https://tasmota.github.io/docs/LoRa-and-LoRaWan-Bridge/#configuration

**Reply by ChrisVonFrosta** [reply]
I am aware of that and initially tried it that way. Since I believed that I had set the GPIOs correctly, I also tried the esp32 version. According to the Online Installer, the esp32-bluetooth (unofficial) build is compatible with the esp32-S3. I mistakenly believed that all 32 versions supported the S3.

The documentation page states that LoRa is included in the esp32 image. Hence, my attempt with esp32.

I've also considered incorrect GPIO settings. I just can't see what's missing. I compared it with the T-Watch, which I have running here with LoRa, but with different GPIOs.

What am I overlooking? What I noticed is that DIO0 is supposed to be on GPIO 33, and GPIO 33 is set to "none" by default in all images.

**Comment by ChrisVonFrosta** [comment]
ok got it.  DO1 is not shown on product info page but i found the schamtics on git.
So Lora_busy is GPIO 36 and DIO1 is on GPIO 9. 
After LoRa Busy is set, the module is initialized. 
Thank you!
